src/html-media-source.js

```diff
@@ -159,6 +159,18 @@ export default class HtmlMediaSource extends videojs.EventTarget {
       });
     };
 
+    this.onPlayerSeeking_ = (event) => {
+      this.sourceBuffers.forEach((sourceBuffer) => {
+        if (sourceBuffer.transmuxer_) {
+          let range = this.player_.buffered();
+          let current = this.player_.currentTime();
+          if (current >= range.end(0) || current <= range.start(0)) {
```

Comment:
You should check `range.length` before attempting to call `start` or `end` because if the buffer is empty you'll get an error